net/models.py

```diff
@@ -84,6 +84,12 @@ def link_to_peeringdb(self):
         any other cases `None` will be returned. The value will also be saved in the
         corresponding field of the model.
         """
+
+        # If data imported from PDB is null+null then there the query
+        # could return many objects and will runtime.
+        if self.ipv4_address is None and self.ipv4_address is None:
```

Comment:
IPv4 referenced two times, probably a copy/paste issue :)

Reply:
You spotted it the same time I did — another push coming :)